test/Microsoft.Buld.Sql.Tests/TestUtils.cs

```diff
@@ -9,27 +9,38 @@ namespace Microsoft.Build.Sql.Tests
 {
     public static class TestUtils
     {
+        private const string DotnetToolPathEnvironmentVariable = "DOTNET_TOOL_PATH";
+
         /// <summary>
         /// Returns the full path to the dotnet executable based on the current operating system.
+        /// Path to dotnet tool can be set by build pipelines via DotnetToolPathEnvironmentVariable.
         /// </summary>
         public static string GetDotnetPath()
         {
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-            {
-                return @"C:\Program Files\dotnet\dotnet.exe";
-            }
-            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
-            {
-                return "/usr/bin/dotnet/dotnet";
-            }
-            else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
-            {
-                return "/usr/local/share/dotnet/dotnet";
-            }
-            else
+            string dotnetExecutable = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "dotnet.exe" : "dotnet";
+            string? dotnetPath = Environment.GetEnvironmentVariable(DotnetToolPathEnvironmentVariable);
+            if (string.IsNullOrEmpty(dotnetPath))
             {
-                throw new NotSupportedException("Tests are currently not supported on " + RuntimeInformation.OSDescription);
+                // Determine OS specific dotnet installation path
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    dotnetPath = @"C:\Program Files\dotnet";
+                }
+                else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                {
+                    dotnetPath = "/usr/bin/dotnet";
+                }
+                else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                {
+                    dotnetPath = "/usr/local/share/dotnet";
+                }
```

Comment:
I'm surprised there isn't a way built into dotnet to get this info, but I can't find anything about it!